### DISCUSSION TITLE
2006/07/18/labview-%E7%A8%8B%E5%BA%8F%E7%9A%84%E5%86%85%E5%AD%98%E4%BC%98%E5%8C%96-1

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#94] [Announcements]
# 2006/07/18/labview-%E7%A8%8B%E5%BA%8F%E7%9A%84%E5%86%85%E5%AD%98%E4%BC%98%E5%8C%96-1

全书： https://labview.qizhen.xyz/

https://blog.qizhen.xyz/2006/07/18/labview-%E7%A8%8B%E5%BA%8F%E7%9A%84%E5%86%85%E5%AD%98%E4%BC%98%E5%8C%96-1

**Comment by ruanqizhen** [comment]
之前博客下的留言：


Qizhen
2009年03月11日 17:10 编辑
to博, 我没有使用过labsql工具包，所以没法回答你的问题了。可以去论坛发贴询问一下。

回复
Qizhen
2009年03月11日 17:09 编辑
最近是比较忙，不会每天上网。所以回答问题可能较慢。子VI如果设计的好，是不会复制传入的数据的。所以我才建议多使用子VI。只要VI设计合理，就可以又用子vi，又不添加内存消耗。LabVIEW是自动释放内存的，但不是在内存没用后，就立即释放。释放内存也是会占用一定CPU资源的，所以LabVIEW会在自认为最合理的时候才释放内存。调用这个Request deallocation函数，可以立即释放所有不需要的内存。

回复
gufei
2009年03月10日 15:05 编辑
阮老师最近不在嘛？我的问题是期待您的回答呢：）你的这句话“第一，把一个复杂VI分解为数个子VI”，我有点疑问，子VI一般是不是都是要复制传入数据的？我想应该是的，那么子vi每次都要复制一份数据，那不是得不偿失？我前几天当面问了您的同事，他说对于大数据，如果在子vi的代码本身对数据只是简单处理，本身没有复制，那么就不要用子vi,不知道这个说法对不对？那么我如何又想用子vi，又不添加内存消耗呢？您也知道在一个复杂处理的处理中，接线器的方法不一定能用上。还有那个Request deallocation 函数是干什么的？子vi不都是自动释放内存嘛？ 急切期待您的解答！！！麻烦您了

回复
博
2006年11月2日 19:45 编辑
一看就知道阮大侠是高手，我接触labview不久，我想请教您有关labview里面处理数据库的问题，我用的是sqlserver，想进行数据库压缩，不知道可以不？我调用labsql工具包里的函数好象解决不了，非常感谢！

回复
Qizhen
2006年07月19日 18:28 编辑
另开一贴讨论如何使用 VI 的重入属性（Reentrant）

回复
竹贤
2006年07月18日 18:06 编辑
请问，在什么情况下需要设置子VI的重入（Reentrant）属性？

回复

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2009年06月11日 08:56 编辑
哪个逻辑，能说详细点不？

回复
shihua
2009年06月10日 22:36 编辑
罗技写错了。请修改

回复
gufei
2009年03月2日 14:14 编辑
这几篇我看了一遍又一遍，收获很大 ，但是还是没能灵活应用到我的项目：）还要继续努力

回复
monny
2007年08月18日 11:34 编辑
你好啊  谢谢你的加为好友啊`~~看来你是个电脑高手呢 以后多多指教啊`!!!!祝你 好

回复
vi
2007年08月18日 01:12 编辑
咯….有够让我无语的… 

回复
